packages/main/src/themes/Badge.css

```diff
@@ -132,3 +132,20 @@
 	border-color: var(--ui5-badge-color-scheme-10-border);
 	color: var(--ui5-badge-color-scheme-10-color);
 }
+
+/* ---Slotted Badges--- */
```

Comment:
Hi @stbodurov everything looks good,one thing that we need to check is if these styles can be moved to Avatar.css where they logically belong



```.css
/* ---Slotted Badges--- */
/* Avatar Badge */
:host([slot="badge"][avatar-badge]) {
	padding: 6%;
 }
/* Make icon take full width minus 6% padding
 as it is not possible to access the size attribute
 of ui5-badge without :host-context() */
:host([slot="badge"][avatar-badge]) ::slotted([ui5-icon][slot="icon"]) {
	width: 100%;
	height: 100%;
	min-width: 100%;
	min-height: 100%;
}
```

I suppose this selector is the reason to add them in the Badge.css file

```css
:host([slot="badge"][avatar-badge]) ::slotted([ui5-icon][slot="icon"]) {
}
```
Perhaps, these styles need to be in the Badge Shadow Dom, but we can try to make it work with parts.
If we can, better we move these styles to Avatar.css


BR,
ilhan